neurogym/utils/data.py

```diff
@@ -55,12 +55,9 @@ def __init__(
             # TODO: infer sequence length from task
             seq_len = 1000
 
-        obs_shape = env.observation_space.shape
-        action_shape = env.action_space.shape
-        if len(action_shape) == 0:
-            self._expand_action = True
-        else:
-            self._expand_action = False
+        obs_shape = tuple(env.observation_space.shape)  # type: ignore[arg-type]
+        action_shape = tuple(env.action_space.shape)  # type: ignore[arg-type]
```

Comment:
see [comment](https://github.com/neurogym/neurogym/pull/145/files#r1993239624) in PR #145.

This workaround is also different from the one in 145, so ultimately it would lead to conflicts. Better to just solve it properly once.

Reply:
Sync'd with #145